src/vos/ilog_internal.h

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
Not opposed to this change necessarily but this is only used by ilog.c, right?

Reply:
No. It was necessary to write a unit test for newly created `ilog_is_valid()`. Please see `src/vos/tests/vts_ilog.c`.